2017/test/ColorTest.java

```diff
@@ -0,0 +1,56 @@
+package test;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+import com.qualcomm.robotcore.hardware.ColorSensor;
+import com.qualcomm.robotcore.hardware.DeviceInterfaceModule;
+import com.qualcomm.robotcore.hardware.DigitalChannelController;
+import com.qualcomm.robotcore.hardware.Servo;
+
+import java.util.ServiceConfigurationError;
+
+import opmodes.NeverlandServoConstants;
+import team25core.ColorSensorTask;
+import team25core.Robot;
+import team25core.RobotEvent;
+
+/**
+ * Created by Izzie on 2/10/2016.
+ */
+@Autonomous(name = "TEST Color", group="AutoTeam25")
+public class ColorTest extends Robot {
```

Comment:
I think I see what you are doing, copying tests from 2016, and slightly modifying to make more generic.  For tests like this that may carry over from year to year perhaps we want to create a java/tests directory (next to team25core), and maybe we move generic examples into java/examples, instead of 2017/examples?   Thinking down the road...